services/venice-controller/src/main/java/com/linkedin/venice/controller/StoreBackupVersionCleanupService.java

```diff
@@ -60,26 +86,35 @@ protected StoreBackupVersionCleanupService(
     this.sleepInterval = TimeUnit.MINUTES.toMillis(5);
     this.defaultBackupVersionRetentionMs = multiClusterConfig.getBackupVersionDefaultRetentionMs();
     this.time = time;
+    this.httpAsyncClient = HttpAsyncClients.custom()
```

Comment:
Shall we only spin up the http client when this feature is enabled?

Reply:
Currently the config is multi-cluster. So spinup could not be controlled for all unless we make it a cluster->client map or single config. Do you think spinning up a client is expensie?

Reply:
If this service is shared by multiple clusters, it is fine to spin it up regardless.
The cost should be low when not being used.